### DISCUSSION TITLE
Änderung "Web API Endpoint" Auswirkungen auf MQTT und Home Assistant?

### DISCUSSION THREAD

**Discussion by NintendoTito** [tbnobody/OpenDTU#1743] [Q&A] [answered]
Hallo zusammen!

Ich habe gesehen, dass es im aktuellen Update 24.2.12 zu einigen "Breaking Change" kommt:

> * 1b637f08      2024-01-30      BREAKING CHANGE: Web API Endpoint /api/livedata/status and /api/prometheus/metrics
> * e1564780      2024-01-30      BREAKING CHANGE: Web API Endpoint /api/livedata/status and /api/prometheus/metrics
> * f0b5542c      2024-01-30      BREAKING CHANGE: Web API Endpoint /api/livedata/status and /api/prometheus/metrics
> * c27ecc36      2024-01-29      BREAKING CHANGE: Web API Endpoint /api/livedata/status

Wird diese Änderungen/Verschiebungen der Werte in andere Sektionen dazu führen, dass ich MQTT bzw. die Daten im Home Assistant neu zuordnen bzw. anpassen muss?

**Comment by Juergen2453** [comment, marked as answer]
Nur bei der API hat sich etwas geändert.
Bei MQTT hat sich nichts geändert.

**Reply by NintendoTito** [reply]
Ah, ok, danke. Sorry, das heraus zu lesen, dafür reichen meine Kenntnisse nicht... 😬
